ansible/roles/screenly/files/screenly_utils.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+recover(){
+	sudo service screenly-viewer stop
+	rm -rf /home/pi/.screenly /home/pi/screenly_assets
+	tar -xvf "$1" -C /home/pi/
```

Comment:
Yeah, I noticed. However, given the magnitude of what we're doing here, it doesn't hurt with an extra level of security in case someone is trying to use the script directly.